changelog_unreleased/javascript/16563.md

```diff
@@ -0,0 +1,21 @@
+#### Don't remove useless `\` in string literals (#16563 by @sosukesuzuki)
+
+Previously, Prettier would remove useless escape characters (`\`) from string literals. However, this behavior was inconsistent as it did not apply to template literals, which was reported in [issue #16542](https://github.com/prettier/prettier/issues/16542).
+
+This issue is a feature request to extend this behavior to template literals as well.
+
+After discussing this internally, the Prettier team concluded that removing useless escape characters, whether in string literals or template literals, is the responsibility of a linter, not a formatter.
+
+This change disables the removal of useless escape characters in string literals across all languages supported by Prettier. To keep the previous behavior, we recommend using ESLint rules like [`no-useless-escape`](https://eslint.org/docs/latest/rules/no-useless-escape).
```

Comment:
Maybe we should mention that we still remove `\` before quotes, [playground](https://deploy-preview-16563--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEBDAvAHRJ3PMDk2IANCBAA4wCW0AzsqKgE7MQDuACiwgyqgBt2qAJ4MyAI2aowAazgwAyqgC2cADLUocZADNBdOJOlyFiijK0BzZDGYBXIyEMrqth07gAPCnGbU1WEEAFT8oFmo4Pn0BQzI6awE4AEV7CHg9AycAKzovRUSUtIykGLiQAEdiuE42Cj40OgBabTgAE3bSEDtUagFrAGEIFRVUZDQBAS6EqCskgEEYO2oJe3hOP01tTNinAAsYFQEAdT3qeDoLMDhFXnPqADdzkXGwOnEQB8cASSgO2EUYH8VHmf0UMBESR25QobEMx2kFHGsKifgeOjIWkMzBgtVQVlG0KcFmY2PGElQEjgUzIsK0MGO1DaMD2yAAHAAGMjMOBVag8vEEsalLJkGCUxnM1lIABMZHshmClOiopAcBUVLaHTa6lQs3s+LgADEIMxRktrONUGsICAAL52oA), I think the behavior is expected, otherwise the output will add `\` when switching `singleQuote` between `true` and `false` since we can't tell if `\` is inserted by us.